docs/BUILD.bazel

```diff
@@ -3,7 +3,7 @@
 load("@aspect_bazel_lib//lib:docs.bzl", "stardoc_with_diff_test", "update_docs")
 
 stardoc_with_diff_test(
-    name = "rules",
+    name = "jasmine_test",
     bzl_library_target = "//jasmine:defs",
```

Comment:
Labelling all "defs" as "jasmine_test" seems odd but I guess we can clean that up if we ever add more rules 🤷 

Same thing with jest and I'm sure others that have a simple enough defs.bzl that everything is in it.

Reply:
we can document a subset of the symbols in `defs.bzl` like https://github.com/bazel-contrib/rules_oci/blob/main/docs/BUILD.bazel#L5-L12

Reply:
I changed this to only document `jasmine_test` and not the underlying `jasmine_test_rule`